internal/store/store.go

```diff
@@ -560,11 +560,11 @@ func (s *Store) getImageBuildDirectory(composeID uuid.UUID, imageBuildID int) st
 	return fmt.Sprintf("%s/%d", s.getComposeDirectory(composeID), imageBuildID)
 }
 
-func (s *Store) PushCompose(distro distro.Distro, composeID uuid.UUID, bp *blueprint.Blueprint, repos []rpmmd.RepoConfig, packages, buildPackages []rpmmd.PackageSpec, arch, composeType string, size uint64, uploadTarget *target.Target) error {
```

Comment:
The `distro` and `arch` parameters are no longer used, I suggest removing them in this PR.

Reply:
I'd like to do clean-ups separately. This is massive as it is, but was intended to do the minimal amount of changes in order to replace the interfaces. Next I get to do the immensely satisfying task of deleting code. (But yes, I agree, these should be dropped from here).